tests/integ/scala/test_dataframe_reader_suite.py

```diff
@@ -332,6 +332,30 @@ def test_to_read_files_from_stage(session, resources_path, mode):
         session.sql(f"DROP STAGE IF EXISTS {data_files_stage}")
 
 
+@pytest.mark.parametrize("mode", ["select", "copy"])
+def test_to_read_file_from_local(session, resources_path, mode):
+
+    test_files = TestFiles(resources_path)
+
+    reader = get_reader(session, mode)
+
+    try:
+        df = (
+            reader.schema(user_schema)
+            .option("compression", "auto")
+            .csv(f"{test_files.test_file_csv}")
+        )
+        res = df.collect()
```

Comment:
I know previously we wrote tests like this and we should fix it. We already have `Utils.check_answer` and you can use this for assertion (see how other tests use this method)